packages/elements/package.json

```diff
@@ -28,7 +28,8 @@
     "deploy": "gh-pages -d example/build"
   },
   "dependencies": {
-    "@stenajs-webui/core": "0.0.51"
+    "@stenajs-webui/core": "0.0.51",
+    "react-spring": "^8.0.27"
```

Comment:
Do we need this, instead of CSS animations?

Reply:
that's fair, will try